### PR TITLE
Feature/destruction models

### PR DESCRIPTION
**Pull request by annashamray**
Fixes https://github.com/GemeenteUtrecht/record-management-app/issues/2

### CODE REVIEW

File: src/rma/destruction/admin.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+
+class DestructionListReviewInline(admin.TabularInline):
+    model = DestructionListReview
+    fields = ("author", "text", "status")
+    extra = 1
+
+
+@admin.register(DestructionList)
+class DestructionListAdmin(FSMTransitionMixin, admin.ModelAdmin):
```
Comment (sergei-maertens) [inline src/rma/destruction/admin.py:27]: please add:

* `search_fields`: name
* `date_hierarchy`: created
* `list_filter`: status

Comment (sergei-maertens) [inline src/rma/destruction/admin.py:27]: Please make the `author` and `assignee` `raw_id_fields`

File: src/rma/destruction/admin.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    list_display = ("name", "author", "assignee", "status")
+    readonly_fields = ("status",)
+    fsm_field = ["status"]
+    inlines = [DestructionListItemInline, DestructionListReviewInline]
+
+
+@admin.register(DestructionListItem)
+class DestructionListItemAdmin(FSMTransitionMixin, admin.ModelAdmin):
```
Comment (sergei-maertens) [inline src/rma/destruction/admin.py:35]: please add:

* `search_fields`: zaak
* `list_filter`: status

File: src/rma/destruction/admin.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+
+class DestructionListItemReviewInline(admin.TabularInline):
+    model = DestructionListItemReview
+    fields = ("destruction_list_item", "text", "suggestion")
+    extra = 1
+
+
+@admin.register(DestructionListReview)
+class DestructionListReviewAdmin(admin.ModelAdmin):
```
Comment (sergei-maertens) [inline src/rma/destruction/admin.py:48]: - please set `destruction_list` and `author` fields in `raw_id_fields`
- please set `date_hierarchy` to `created`

File: src/rma/destruction/admin.py [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+    list_display = ("destruction_list", "author")
+    inlines = [DestructionListItemReviewInline]
+
+
+@admin.register(DestructionListItemReview)
+class DestructionListItemReviewAdmin(admin.ModelAdmin):
```
Comment (sergei-maertens) [inline src/rma/destruction/admin.py:54]: please add:

- `raw_id_fields`: `destruction_list_review` and `destruction_list_item`
- `list_filter`: suggestion
- `search_fields`: `destruction_list_item__zaak`

File: src/rma/destruction/admin.py (hunk 1 above)
Comment (annashamray) [reply src/rma/destruction/admin.py:27]: added

Comment (annashamray) [reply src/rma/destruction/admin.py:27]: `raw_id_fields` are added

File: src/rma/destruction/admin.py (hunk 2 above)
Comment (annashamray) [reply src/rma/destruction/admin.py:39]: added

File: src/rma/destruction/admin.py (hunk 3 above)
Comment (annashamray) [reply src/rma/destruction/admin.py:55]: done

File: src/rma/destruction/admin.py (hunk 4 above)
Comment (annashamray) [reply src/rma/destruction/admin.py:63]: done

**Pull request merged**